.github/workflows/deploy-gh-pages-explorer.yml

```diff
@@ -1,12 +1,12 @@
 # Simple workflow for deploying static content to GitHub Pages
-name: Deploy to GH Pages
+name: Deploy explorer to GH Pages
 
 on:
   workflow_dispatch:
 
 env:
-  VITE_BACKEND_URL: https://backend-v113.testnet.alephium.org
-  VITE_NODE_URL: https://wallet-v20.testnet.alephium.org
+  VITE_BACKEND_URL: https://backend.testnet.alephium.org
+  VITE_NODE_URL: https://wallet.testnet.alephium.org
```

Comment:
Isn't it safe to always use the [latest aliases](https://docs.alephium.org/dapps/public-services#api-aliases)?

Reply:
Mhhh, I think it's ok if the testnet deploy fails, but I would use fixed version for the mainnet version.